.github/workflows/integration.yml

```diff
@@ -0,0 +1,39 @@
+name: Flutter Gallery Integration Tests
```

Comment:
This seems unrelated to golden tests, can you update the PR description to include this too (or split into a separate PR if you want)

Reply:
Updated the PR description. I just moved some content to another file.